spec/support/core_helpers.rb

```diff
@@ -44,4 +44,18 @@ def supports_value_expectations?
       false
     end
   end
+
+  module ClassMethods
+    def integration_test
```

Comment:
`integration_test` imho is not very clear what it does at the top of a test. May I suggest something along the lines of `skip_unless_integration_testing_enabled` or something along does lines?

Reply:
It's declarative instead of imperative :)

Renamed.